src/services/filters-parser.js

```diff
@@ -0,0 +1,131 @@
+import _ from 'lodash';
+import Interface, { BaseFiltersParser } from 'forest-express';
+import OperatorDateIntervalParser from './operator-date-interval-parser';
+import { NoMatchingOperatorError, InvalidFiltersFormatError } from './errors';
+import utils from '../utils/schema';
+
+const AGGREGATOR_OPERATORS = ['and', 'or'];
+
+function FiltersParser(model, timezone, options) {
+  this.operatorDateIntervalParser = new OperatorDateIntervalParser(timezone);
+  const schema = Interface.Schemas.schemas[utils.getModelName(model)];
+
+  this.perform = filtersString => BaseFiltersParser
+    .perform(filtersString, this.formatAggregation, this.formatCondition);
+
+  this.formatAggregation = (aggregator, formatedConditions) => {
+    const aggregatorOperator = this.formatAggregatorOperator(aggregator);
+    return { [aggregatorOperator]: formatedConditions };
+  };
+
+  this.formatCondition = (condition) => {
+    if (!condition) { throw new InvalidFiltersFormatError('No conditions provided'); }
+    if (!condition.field) { throw new InvalidFiltersFormatError('No field provided'); }
```

Comment:
You did not check if it is an object, nor if it has an operator and a value keys